.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -43,17 +43,47 @@ If you need to submit a pr for any of these repos during this period, you should
 
 - [ ] **Enable code freeze bot** In order to enable the code freeze GitHub you need to change the line `if: github.repository_owner == 'adoptium' && false` to be `if: github.repository_owner == 'adoptium' && true` in the [code-freeze.yml](https://github.com/adoptium/.github/blob/main/.github/workflows/code-freeze.yml#L21) GitHub workflow. Please contact the PMC if you need help merging this change.
 - [ ] **Disable nightly testing** to free up resources and ensure no competing jobs during release week
-- [ ] **Update aqaReference** to update with new git tag for 'aqaReference' to be used in release pipeline
+- [ ] Tag the build repositories (temurin-build, ci-jenkins-pipelines, jenkins-helper) for the release level tooling: github/\<repo\>/releases, "Draft new release". Choose tag: "v\<YY\>.\<MM\>.01" (increment 01 if needed..)
+- [ ] **Update aqaReference** to update with new git branch for 'aqaReference' to be used in release pipeline: https://github.com/adoptium/aqa-tests/branches (branch name to use should match the name of the [latest aqa-tests release](https://github.com/adoptium/aqa-tests/releases/latest))
+- [ ] Generate release pipeline jobs for the new build scripts repo release tag, helper repo releasetag and aqaReference: https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/release-build-pipeline-generator/build?delay=0sec
+- [ ] TC: Run the ProcessCheckMultiNode process cleaning job on all ci.role.test nodes, to ensure healthy state, verify all nodes successful: https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/build?delay=0sec
+- [ ] TC: Run the Setup_JCK_Run_Multinode job with CLEAN_DIR=true (to purge any old release contents/results) on all ci.role.test nodes, this will extract the jck_run folder with all the temurin.jtx exclude files, verify all nodes successful : https://ci.eclipse.org/temurin-compliance/job/Setup_JCK_Run_Multinode/build?delay=0sec
 - [ ] **Run a trial release pipeline** to ensure less surprises on release day (typically against a milestone build)
 
+<details>
```

Comment:
I'd suggest that this level of detail should perhaps be a reference into the RELEASING.md doc to keep this purely as a "what has been done" doc.

Reply:
yep: https://github.com/adoptium/temurin-build/blob/master/RELEASING.md#auto-way---before-release-week-auto-test
(I need to update that a bit..)